.github/dependabot.yml

```diff
@@ -0,0 +1,162 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  # Configuration for Maven
+  - package-ecosystem: "maven" 
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
+    target-branch: "master"
+    ignore:
+      # Ignore major and minor version updates for dependencies with group IDs starting with jakarta
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
+
+      # Ignore major and minor version updates for dependencies with group IDs starting with org.eclipse.microprofile
+      - dependency-name: org.eclipse.microprofile*"
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
+
+      # Ignore major version updates for dependencies with group ID starting with fish.payara
+      - dependency-name: "fish.payara.*"
+        update-types: ["version-update:semver-major"]
+
+      # Ignore major version updates for individual dependencies within the org.eclipse.persistence group
+      - dependency-name: "org.eclipse.persistence:org.eclipse.persistence.core"
```

Comment:
These dependencies (e.g org.eclipse.persistence:org.eclipse.persistence) not required for starter.